### DISCUSSION TITLE
Filter Issues by Milestones

### DISCUSSION THREAD

**Discussion by crivchri** [community/community#3853] [Mobile]
Hi,

Could you please add the function to filter the Issues by Milestones (as on the website)?

Thanks and kind regards

**Comment by nplasterer** [comment]
Thanks for sharing your feedback. This is something we’re actively working on. Stay tuned! 

If you want to be one of the first people to try it out when it gets released. You can join our Beta program. 
For Android, users can join our Beta program via [this link](https://play.google.com/apps/testing/com.github.android).
For iOS, users can join our testflight via [this link](https://testflight.apple.com/join/NLskzwi5).
